.eslintrc

```diff
@@ -5,20 +5,24 @@
     "./node_modules/@theforeman/vendor-dev/eslint.extends.js"
   ],
   "rules": {
-    "prettier/prettier": ["error", {
-      "singleQuote": true,
-      "trailingComma": "es5"
-    }],
-    "import/no-unresolved": ["error", {
-      "ignore": ['foremanReact/.*']
-    }],
-    "import/extensions": ["error", {
-      "ignore": ['foremanReact/.*']
-    }]
-  },
-  "settings": {
```

Comment:
do we need to update `eslint-plugin`?

Reply:
No, just my editor decided to run prettier on this file